index.html

```diff
@@ -1169,19 +1169,77 @@ <h2>Relative DID URLs</h2>
 
   </section>
 
-  <section class="informative">
+  <section>
     <h1>Data Model</h1>
-
+    <p>
+This specification defines an abstract data model for <a>DID documents</a>,
+independent of any specific representation. This section provides a
+high-level description of the data model, a set of requirements for
+representations, and a set of requirements for extensibility.
+    </p>
     <section>
       <h2>Definition</h2>
+      <p>
+A <a>DID document</a> consists of a collection of name-value pairs which will be
+referred to as properties in this document. The definitions of each of these
+properties are specified in section <a href="#core-properties"></a>. Specific
+representations are defined in section <a href="#core-representations"></a>.
+      </p>
     </section>
-
     <section>
-      <h2>Extensibility</h2>
+      <h2>Representations</h2>
+      <p>
+Following are the requirements for conformant representations.
+      </p>
+      <ol>
+        <li>
+A representation MUST define an unambiguous encoding and decoding of all
+properties defined in this specification. This definition MUST enable lossless
+conversion of the value of any property into another representation. This means
+anything you can represent in the DID document model can be represented in any
+compliant representation.
+        </li>
+        <li>
+The representation MUST have an unambiguous MIME type.
+        </li>
+        <li>
+The representation MUST define fragment processing rules for its MIME type that
+are conformant with the fragment processing rules defined in section
+<a href="#fragment"></a> of this specification.
+        </li>
+      </ol>
+        <p>
+The core representations are specified in section <a href="#core-representations"></a>.
+        </p>
     </section>
-
     <section>
-      <h2>Representation Requirements</h2>
+      <h2>Extensibility</h2>
+      <p>
+The data model supports two types of extensibility.
+      </p>
+      <ol>
+        <li>
+For maximum interoperability, it is RECOMMENDED that extensions use the official
+W3C registries defined by the DID Core and Extensions Registries specification
+[[DID-CORE-REGISTRIES]]. The use of this registry for new properties or other
+extensions is the only specified method for two independent implementations to
+be able to work together. Representations MUST process any extension in the DID
+Core and Extensions Registries according to the definition in the registry.
+Registry-based extensions MUST enable lossless conversion between all conformant
+representations.
+        </li>
+      <li>
+Representations MAY define other extensibility mechanisms including methods for
+decentralized extensions. Such extension mechanisms SHOULD support lossless
+conversion into any other conformant representation.
```

Comment:
```suggestion
decentralized extensions. Such extension mechanisms MUST support lossless
conversion into any other conformant representation.
```

It's MUST in the paragraphs above... downgrading this to a SHOULD here enables the introduction of values that can't be represented in all representations, harming interop. We should be consistent... either MUST in both places, or SHOULD in both places. I'd prefer MUST in both places.

Reply:
Agreed it should be MUST in both places.